centreon/src/Core/Dashboard/Infrastructure/API/ShareDashboard/ShareDashboardController.php

```diff
@@ -25,54 +25,27 @@
 
 use Centreon\Application\Controller\AbstractController;
 use Core\Dashboard\Application\UseCase\ShareDashboard\ShareDashboard;
-use Core\Dashboard\Application\UseCase\ShareDashboard\ShareDashboardRequest;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-
+use Symfony\Component\HttpKernel\Attribute\MapRequestPayload;
+use Symfony\Component\Security\Http\Attribute\IsGranted;
+
+#[IsGranted(
+    'dashboard_access_editor',
```

Comment:
```suggestion
    DashboardVoter::DASHBOARD_ACCESS_EDITOR
```
You can resolve if you don't want to apply

Reply:
Noted I'll apply this on futur PR